rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -16,44 +16,262 @@
     private final static int TIMEOUT_IN_MS = 10000;
     private long rtmpPointer = 0;
 
+    private int rtmpError = RtmpIOException.RTMP_ERROR_NONE;
+
     /** Socket send timeout value in milliseconds */
     private int sendTimeoutInMs = TIMEOUT_IN_MS;
     /** Socket receive timeout value in seconds */
     private int receiveTimeoutInMs = TIMEOUT_IN_MS;
 
     public static class RtmpIOException extends IOException {
 
+        /**
+         * No error
+         */
+        public final static int RTMP_ERROR_NONE = 0;
```

Comment:
It's confusing to define non-error values inside `RtmpIOException`. I think it makes sense to move this one and the one below into the outer class, and to make them private. Maybe `RTMP_ERROR_NONE` should be renamed to `RTMP_SUCCESS`, since it's not really an error code.

Reply:
Done.